src/web/topic/components/TopicPane/TopicPane.tsx

```diff
@@ -11,76 +13,136 @@ import {
+    return (
+      <TopicDrawer
+        anchor={isLandscape ? "left" : "bottom"}
+        isOpen={isTopicPaneOpen}
+        onToggleClick={handleToggle}
+      >
+        <TabContext value={selectedTab}>
+          <TabList
+            onChange={(_, newValue: "Details" | "Views") => setSelectedTab(newValue)}
+            centered
+          >
+            <Tab label="Details" value="Details" />
+            <Tab label="Views" value="Views" />
+          </TabList>
+          <TabPanel value="Views">
+            <TopicViews />
+          </TabPanel>
+          <TabPanel value="Details">
+            {selectedGraphPart !== null ? (
+              // Could consider using a `viewingGraphPart` separate from selected so that graph
+              // part details for parts outside of the shown diagram can be displayed, without
+              // losing tracking of the selected part per diagram.
+              <GraphPartDetails graphPart={selectedGraphPart} key={selectedGraphPart.id} />
+            ) : (
+              <TopicDetails />
+            )}
+          </TabPanel>
+        </TabContext>
+      </TopicDrawer>
+    );
+  }
 
   return (
     <>
-      {/* div to enable menu button to be positioned to the right of the pane */}
-      <PositionedDiv>
-        <TogglePaneButton onClick={handlePaneToggle} color="primary" isLandscape={isLandscape}>
-          <ToggleIcon />
-        </TogglePaneButton>
-        {/* `permanent` because `persistent` adds transitions that conflict with our styles */}
-        <StyledDrawer
-          variant="permanent"
-          open={isTopicPaneOpen}
-          anchor={isLandscape ? "left" : "bottom"}
-          isLandscape={isLandscape}
-        >
-          <TabContext value={selectedTab}>
-            <TabList
-              onChange={(_, newValue: "Details" | "Views") => setSelectedTab(newValue)}
-              centered
-            >
-              <Tab label="Details" value="Details" />
-              <Tab label="Views" value="Views" />
-            </TabList>
-
-            <TabPanel value="Details">
-              {selectedGraphPart !== null ? (
-                // Key ensures that details components re-render, re-setting default values.
-                // Could consider using a `viewingGraphPart` separate from selected so that graph
-                // part details for parts outside of the shown diagram can be displayed, without
-                // losing tracking of the selected part per diagram.
-                <GraphPartDetails graphPart={selectedGraphPart} key={selectedGraphPart.id} />
-              ) : (
-                <TopicDetails />
-              )}
-            </TabPanel>
-
-            <TabPanel value="Views">
-              <TopicViews />
-            </TabPanel>
-          </TabContext>
-        </StyledDrawer>
-      </PositionedDiv>
+      <TopicDrawer anchor="left" isOpen={isDetailsOpen} onToggleClick={toggleDetails}>
+        <TabContext value="Details">
```

Comment:
I actually think I like `Views` on the left and `Details` on the right, because I think `Details` will change more often (e.g. every time a different node is selected) and for some reason I feel like the left/flex-start side makes sense to be more stable.